testing/upload_validation.ts

```diff
@@ -1178,6 +1180,7 @@ const baseSyncTableSchema = {
   getter: syncFormulaSchema,
   entityName: z.string().optional(),
   defaultAddDynamicColumns: z.boolean().optional(),
+  propertyAutocomplete: objectPackFormulaSchema.optional(),
```

Comment:
It works, and it's based on what's done for the `getSchema` metadata formula (schema is optional for `objectPackFormulaSchema`). Some relevant bits of code from the `getSchema` case:

```
  getSchema: formulaMetadataSchema.optional(),


const formulaMetadataSchema = z
  .union([numericPackFormulaSchema, stringPackFormulaSchema, booleanPackFormulaSchema, objectPackFormulaSchema])

const objectPackFormulaSchema = zodCompleteObject<Omit<ObjectPackFormula<any, any>, 'execute'>>({
  ...commonPackFormulaSchema,
  resultType: zodDiscriminant(Type.object),
  // TODO(jonathan): See if we should really allow this. The SDK right now explicitly tolerates an undefined
  // schema for objects, but that doesn't seem like a use case we actually want to support.
  schema: z.union([genericObjectSchema, arrayPropertySchema]).optional(),
});
```

Reply:
Ah interesting, my fault then for leaving this as optional this entire time. Well, cool then, just make sure your case is covered by a unittest so that if this TODO is ever resolved and the schema becomes required for top-level formulas, that it doesn't break autocomplete.

Reply:
Verified that tests in `upload_test.ts` will fail if the schema becomes non-optional in the future.

```
  2) Pack metadata Validation
       Formulas
         sync tables
           valid sync table:
     AssertionError: Expected validation to succeed but failed with error [{"path":"syncTables[0].propertyAutocomplete.schema","message":"Required"}]
      at validateJson (/Users/DavidWeitzman/src/packs-sdk/test/upload_validation_test.ts:56:14)
      at async Context.<anonymous> (/Users/DavidWeitzman/src/packs-sdk/test/upload_validation_test.ts:798:9)
```